src/Controller/Admin/LocationController.php

```diff
@@ -69,7 +67,38 @@ public function newAction(Request $request): Response
     }
 
     /**
-     * Finds and displays an auth entity.
+     * Merge all duplicate location entities.
+     */
+    #[Route('/merge', name: 'merge', methods: ['GET', 'POST'])]
+    public function mergeAction(): Response
```

Comment:
Conceptually, I'm not an enormous fan of this route, as it only supports exact matches. Moreover, it feels like a one-time thing. I'd suggest a form to merge multiple items into a single item. Although it requires more manual labor, it also gives more granularity to the user.

Reply:
you're right, yeet'd the entire merge functionality.